platform/ios/scripts/publish-cn.sh

```diff
@@ -0,0 +1,53 @@
+
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+set -u
+
+function step { >&2 echo -e "\033[1m\033[36m* $@\033[0m"; }
+function finish { >&2 echo -en "\033[0m"; }
+trap finish EXIT
+
+#
+# make the framework
+#
+  
+step "Make the framework"
+export BUILDTYPE=Release
+./platform/ios/scripts/package.sh
+echo "making package"
+wait
+echo "finished!"
+
+#
+# zip
+#
+cd build/ios/pkg
+ZIP="mapbox-ios-sdk-cn.zip"
```

Comment:
This should include a version number, otherwise it’s always overwriting the same file.